make.py

```diff
@@ -131,49 +135,50 @@ def add_disputed(polygon, properties):
 
     for country in countries:
         iso = country['iso']
-        polygon, properties = await load_country(conn, iso=iso)
-        properties.update(country)
-        if properties['name:en'] == 'Sahrawi Arab Democratic Republic':
-            continue
-        print(f'''{iso} : "{properties['name']}" ({properties['name:en']})''')
-        if iso == 'IL':
-            polygon = await remove_area(conn, polygon, golan)
-            west_bank, _ = await get_relation(conn, place="region",
-                                              name="الضفة الغربية")
-            polygon = await remove_area(conn, polygon, west_bank)
-        if iso == 'SY':
-            polygon = await add_area(conn, polygon, golan)
-        if iso == 'SS':
-            sudan, _ = await load_country(conn, iso='SD')  # Sudan
-            polygon = await remove_area(conn, polygon, sudan)
-        if properties['name:en'] == 'Sudan':
-            polygon = await remove_area(conn, polygon, bir_tawil)
-        if iso == 'EG':
-            polygon = await add_area(conn, polygon, bir_tawil)
-        if iso == 'NP':
-            claim, props = await get_relation(conn, type="boundary",
+        if country['admin_level'] != '' and int(country['admin_level']) > 0  and int(country['admin_level']) < 4:
```

Comment:
That may just be a naming issue, but having a `country` with `admin_level==''` seems awkward to me.
Do you have in mind how this can happen?

Reply:
Antarctica doesn't have because it's not a country, or I didn't find it inside osm (tagging problem?)

But in any case this test `country['admin_level'] != ''` can be deleted I think, because I pushed 0 by default in the csv file

Reply:
OK.
Maybe set the `admin_level` value once and for all before the `if`?
```python
admin_level = int(country['admin_level'] or 0)
if 0 < admin_level < 4:
    ….
```